cli/getCache.ts

```diff
@@ -8,5 +8,5 @@ export async function getCache(_cachePath: string): Promise<RollupCache> {
   const defaultCache: RollupCache = { modules: [] };
 
   if (!(await fs.exists(cachePath))) return defaultCache;
-  return fs.readJson(cachePath) as Promise<RollupCache>;
+  return JSON.parse(await Deno.readTextFile(cachePath)) as RollupCache;
```

Comment:
Yikes, this was deprecated?

Reply:
Yup `readJson` and `writeJson` were removed.

Reply:
:( okay, thanks!